lib/ecto/parameterized_type.ex

```diff
@@ -100,7 +100,7 @@ defmodule Ecto.ParameterizedType do
   @callback init(opts :: opts()) :: params()
 
   @callback change(old_value :: any(), new_value :: any(), params :: params()) ::
-              {:ok, new_value :: any()} | {:error, Ecto.Changeset.error()} | :skip
+              {:ok, new_value :: any()} | {:error, opts} | :skip
```

Comment:
```suggestion
              {:ok, new_value :: any()} | {:error, keyword()} | :skip
```

Reply:
Ah, what a mistake … `opts` here is for a different purpose - I made mistake as it's really general name, sorry :-)